grammars/php.cson

```diff
@@ -1269,8 +1276,8 @@
         'begin': '''(?xi)
           (
             \\\\?\\b                                                # Optional root namespace
```

Comment:
Without much worrying about it, `(?<![a-z0-9_\\x{7f}-\\x{7fffffff}])` should be sufficient replacement for `\\b`. If someone can prove, it could potentially be removed or replaced by something simpler.

Reply:
Would `(?<!\\d)` also work?

Reply:
It is supposed to assert that none of the characters usable in identifier come immediately before the start of the pattern. If we want to retain the meaning of the word boundary there, then I believe we cannot use a subset of the pattern I proposed or else it might match in some strange cases.

I don't know how to efficiently prove that any of the subsets would work. One would need to know all the possible patterns that can precede this pattern and then verify that none of them can match the characters at the end which you are trying to omit. Then also the same for cases where no match could have been made and the cursor was simply advanced by one. It is easier to just match a wider range of characters as it definitely will not do any harm.

P.S. In unicode mode of oniguruma `\d` seems to match also other characters besides `[0-9]` outside ASCII, so one should be careful when using it.